client/src/pages/Explorer.js

```diff
@@ -73,47 +98,82 @@ function FavoriteCheckBox() {
 
 const ExplorerPage = () => {
   const classes = useStyles();
+  const [generalData, setGeneralData] = useState({places: [], favorites: []});
+  const citiesUrl = "/api/cities/us/?population=1500000";
+  const userFavoriteCitiesUrl = "/api/users/favoriteCities";
+  useEffect(() => {
+    async function getData(){
+      try {
+        const result1 = await fetch(userFavoriteCitiesUrl, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        })
+        const result2 = await fetch(citiesUrl, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          }
+        })
+        const places = await result2.json();
+        const favorites = await result1.json();
+        userFavoritePlaces = Array.isArray(favorites) ? favorites : [];
+        setGeneralData({
+          places: Array.isArray(places) ? places : [],
+          favorites: Array.isArray(favorites) ? favorites : []
```

Comment:
same comment as above. If we're checking for null && undefined we can just do `places: places || []`